frontend/apps/app/features/projects/pages/ProjectsPage/ProjectsPage.tsx

```diff
@@ -1,21 +1,20 @@
 import { urlgen } from '@/utils/routes'
-import type { FC } from 'react'
-import { EmptyProjectsState } from '../../components'
-import { ClientSearchWrapper } from './ClientSearchWrapper'
+import { EmptyProjectsState } from '../../components/EmptyProjectsState'
 import styles from './ProjectsPage.module.css'
+import { ServerProjectsDataProvider } from './ServerProjectsDataProvider'
 import {
   getCurrentOrganization,
   getUserOrganizations,
 } from './getCurrentOrganization'
 import { getProjects } from './getProjects'
 
-interface ProjectsPageProps {
-  organizationId?: string
-}
+export const runtime = 'edge'
```

Comment:
Why do you use `edge`?

Reply:
Thanks for pointing that out.
 I added the code without much thought, but I understand that it is unnecessary, at least at this time.
I have removed it!
https://github.com/liam-hq/liam/pull/1394/commits/dcd22414266b209fb030183fb72c88b3bd3378f3